rpcs3/Emu/Cell/Modules/sceNp.cpp

```diff
@@ -745,7 +745,8 @@ error_code sceNpBasicSetPresenceDetails(vm::cptr<SceNpBasicPresenceDetails> pres
 
 error_code sceNpBasicSetPresenceDetails2(vm::cptr<SceNpBasicPresenceDetails2> pres, u32 options)
 {
-	sceNp.todo("sceNpBasicSetPresenceDetails2(pres=*0x%x, options=0x%x)", pres, options);
+	sceNp.todo("sceNpBasicSetPresenceDetails2(pres=*0x%x, options=0x%x) state=0x%X title=%s status=%s comment=%s data=%s",
+	    pres, options, pres->state, pres->title, pres->status, pres->comment, pres->data);
```

Comment:
possible nullptr deref of pres will crash the emulator. this needs to be either a ternary with nullcheck or logged after line 760